packages/webapp/pages/feeds/[slugOrId]/edit.tsx

```diff
@@ -81,7 +81,8 @@ const EditFeedPage = (): ReactElement => {
   const { user } = useAuthContext();
   const { feedSettings } = useFeedSettings({ feedId, enabled: !!feedId });
   const [isPreviewFeedVisible, setPreviewFeedVisible] = useState(true);
-  const isPreviewFeedEnabled = feedSettings?.includeTags?.length >= 1;
+  const isPreviewFeedEnabled =
+    feedSettings?.includeTags && feedSettings.includeTags.length >= 1;
```

Comment:
probably want to do `!!feedSettings?.includeTags` to keep `isPreviewFeedEnabled` boolean always

Reply:
Good catch, keeping it a boolean is a must, or else, the `shouldEvaluate` on `useConditionalFeature` would break because react-query only accepts a Boolean value and if not, it will throw an error and crash the app on prod. Don't ask me how I know 😭